libs/blocks/section-metadata/section-metadata.js

```diff
@@ -71,6 +77,38 @@ export const getMetadata = (el) => [...el.childNodes].reduce((rdx, row) => {
   return rdx;
 }, {});
 
+async function createShowMoreButton(section) {
+  const seeMoreText = await replacePlaceholder('see-more-features');
+  const showMoreButton = createTag('div', { class: 'show-more-button' });
+  const button = createTag('button', {}, '');
+  button.innerHTML = seeMoreText;
+
+  const iconSpan = createTag('span', {
+    class: 'show-more-icon',
+    'aria-hidden': 'true',
+  }, `${ADD_MORE_ICON}`);
+  button.appendChild(iconSpan);
+  button.setAttribute('aria-label', seeMoreText);
```

Comment:
This should also work and is a bit more brief and clear. `aria-label` should not be needed on the button, since text is already nested inside.
```suggestion
  const button = createTag('button', {}, seeMoreText);

  const iconSpan = createTag('span', {
    class: 'show-more-icon',
    'aria-hidden': 'true',
  }, `${ADD_MORE_ICON}`);
  button.appendChild(iconSpan);
```